apps/challenges/challenge_notification_util.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+from django.conf import settings
+
+from base.utils import send_email
+from challenges.aws_utils import start_workers
+
+logger = logging.getLogger(__name__)
+
+
+def challenge_start_notifier(sender, instance, field_name, **kwargs):
+    prev = getattr(instance, "_original_{}".format(field_name))
+    curr = getattr(instance, "{}".format(field_name))
+    if prev != curr:
+        challenge = instance
+
+        # Start the challenge worker.
+        response = start_workers([challenge])
+        count, failures = response["count"], response["failures"]
+
+        if(count != 1):
+            logger.warning("Worker for challenge {} couldn't start! Error: {}".format(challenge.id, failures[0]["message"]))
+        else:
+            url = "https://evalai.cloudcv.org/web/challenges/challenge-page/{}".format(challenge.id)
```

Comment:
Instead of hard coding the host name here, can we please move it to settings and use it from there?